packages/widgets/src/health-monitoring/component.tsx

```diff
@@ -0,0 +1,345 @@
+                          {t("widget.healthMonitoring.popover.processor")} {healthInfo.cpuModelName}
+                        </List.Item>
+                        <List.Item className="system-info-version" icon={<IconVersions size="1cqmin" />}>
+                          {t("widget.healthMonitoring.popover.version")} {healthInfo.version}
+                        </List.Item>
+                        <List.Item className="system-info-uptime" icon={<IconClock size="1cqmin" />}>
+                          {formatUptime(healthInfo.uptime, t)}
+                        </List.Item>
+                      </List>
+                    </HoverCard.Dropdown>
+                  </HoverCard>
+                </Box>
+                {options.cpu && (
+                  <Box ref={ref} w="100%" h="100%">
+                    <RingProgress
+                      className="ring-cpu-utilization"
+                      roundCaps
+                      size={ringSize}
+                      thickness={ringThickness}
+                      label={
+                        <Center style={{ flexDirection: "column" }}>
+                          <Text
+                            className="ring-cpu-text"
+                            size="3cqmin"
+                          >{`${healthInfo.cpuUtilization.toFixed(2)}%`}</Text>
+                          <HoverCard>
+                            <HoverCard.Target>
+                              <IconCpu className="ring-cpu-icon" size="7cqmin" />
+                            </HoverCard.Target>
+                            <HoverCard.Dropdown>
+                              <List
+                                className="load-average-dropdown-list"
+                                size="sm"
+                                center
+                                spacing="0.5cqmin"
+                                icon={<IconCpu2 size="1cqmin" />}
+                              >
+                                <List.Item className="load-average-title" icon>
+                                  {t("widget.healthMonitoring.popover.loadAverage")}
+                                </List.Item>
+                                <List.Item className="load-average-1min">
+                                  {t("widget.healthMonitoring.popover.oneMinute")} {healthInfo.loadAverage["1min"]}
+                                </List.Item>
+                                <List.Item className="load-average-5min">
+                                  {t("widget.healthMonitoring.popover.fiveMinutes")} {healthInfo.loadAverage["5min"]}
+                                </List.Item>
+                                <List.Item className="load-average-15min">
+                                  {t("widget.healthMonitoring.popover.fifteenMinutes")}{" "}
+                                  {healthInfo.loadAverage["15min"]}
+                                </List.Item>
```

Comment:
After the change in the translation file you could also make something similar to:

```ts
{[1,5,15].map(minutes => (
  <List.Item className={"load-average-${minutes}min"}>
    {t("widget.healthMonitoring.popover.minutes", {count: minutes})}{" "}{healthInfo.loadAverage[`${minutes}min`]} 
 </List.Item>
));
```

Reply:
Because I use singular for minute I don't think I need to use map for 2 items (5, 15)